archive/finder.go

```diff
@@ -63,6 +63,11 @@ func Search(path string, rule Rule, pattern string) (bool, error) {
 		}
 		return false, err
 	}
+	defer finder.Close()
```

Comment:
I looked at other instances of finder.Open(); I see one more case of a missing close in zdx_test.go, in `TestSearch`.